packages/cm6-graphql/package.json

```diff
@@ -15,7 +15,7 @@
   "types": "dist/index.d.ts",
   "sideEffects": false,
   "dependencies": {
-    "graphql-language-service": "^5.1.7"
+    "graphql-language-service": "workspace:"
```

Comment:
where did you find this syntax?

Reply:
not sure what you're speaking of, it's `workspace:` here?
![image](https://github.com/graphql/graphiql/assets/1368727/a8a034df-df32-4f79-8a27-343bd56a6d3d)